src/dpp/channel.cpp

```diff
@@ -87,6 +87,11 @@ channel& channel::set_topic(const std::string& topic) {
 	return *this;
 }
 
+channel& channel::set_guild_id(const snowflake guild_id) {
+	this->guild_id = guild_id;
```

Comment:
the parameter name guild_id shadows the member variable (they have the same name)